src/box/lua/config/instance_config.lua

```diff
@@ -1118,6 +1118,67 @@ return schema.new('instance_config', schema.record({
             end
         end,
     }),
+    security = schema.record({
+        auth_type = schema.enum({
+            'chap-sha1',
+            'pap-sha256',
+        }, {
+            box_cfg = 'auth_type',
+            default = 'chap-sha1',
+            validate = function(auth_type, w)
+                if auth_type ~= 'chap-sha1' and
+                        tarantool.package ~= 'Tarantool Enterprise' then
+                    w.error('chap-sha1 is the only auth_type available in' ..
+                            ' Tarantool CE')
+                end
+            end
+        }),
+        auth_delay = enterprise_edition(schema.scalar({
+            type = 'number',
+            default = 0,
+            box_cfg = 'auth_delay',
+        })),
+        disable_guest = enterprise_edition(schema.scalar({
+            type = 'boolean',
+            default = false,
+            box_cfg = 'disable_guest',
+        })),
+        password_lifetime_days = enterprise_edition(schema.scalar({
```

Comment:
There are quite a few "password" options. What do you think about turning them into `security.password. ...` options? @Lord-KA @Totktonada

Reply:
I think it is best to have structure as similar with box.cfg as possible. "security" itself is a made up name, I don't see the necessity for another one.

Reply:
Well, `security.password.lifetime_days` is not too different from `security.password_lifetime_days` I guess. But it looks more structured.

Reply:
Yea, but `password_lifetime_days` is exactly box.cfg option, while `lifetime_days` isn't.